examples/asteroids/random_network_baseline.py

```diff
@@ -56,7 +56,7 @@
 
 # Excitatory -> readout.
 exc_readout_conn = Connection(source=layers['E'], target=layers['R'], w=0.01 * torch.rand(layers['E'].n, layers['R'].n),
-                              update_rule=Hebbian, nu_pre=1e-2, nu_post=1e-2, norm=0.5 * layers['E'].n)
+                              update_rule=Hebbian, nu=(1e-2, 1e-2), norm=0.5 * layers['E'].n)
```

Comment:
In general, you should use lists, not tuples, so they can be modified at training time.